config/package/tmpfiles.d/uwsgi.conf

```diff
@@ -0,0 +1 @@
+d /run/uwsgi/socorro 0755 socorro nginx -
```

Comment:
The only problem with doing it "right", or at least right-ish, is that we assume everything in `./package/conf` is going into `/etc` in the staged install area, maybe we should make everything in there go into `/` in the staged install area instead.